doc/source/pmm-admin.rst

```diff
@@ -171,30 +178,30 @@ monitoring service parameters along with the :program:`pmm-admin add` command
 when adding the :option:`mongodb:queries` monitoring service.
 
 .. code-block:: bash
-   :caption: Passing an SSL/TLS parameter to :program:`mongod` to enables 
+   :caption: Passing an SSL/TLS parameter to :program:`mongod` to enables
              a TLS connection.
 
-   $ sudo pmm-adming add mongodb:queries -- mongodb.tls
+   $ sudo pmm-adming add mongodb:queries -- --mongodb.tls
```

Comment:
👍

Reply:
So does `mysqld_exporter`, yet you use `--collect.perf_schema.eventsstatements` above.